pages/index.js

```diff
@@ -20,7 +20,7 @@ const Index = ({ membersArr, errorMsg }) => {
 export async function getServerSideProps(context) {
   context.res.setHeader('Cache-Control', `max-age=${CACHE_MAX_AGE}`);
 
-  const membersArr = [];
+  const membersWithImage = [];
```

Comment:
I think this name is not right, are you sure that every member has an image?

Reply:
Right..  I will fix it